app.js

```diff
@@ -38,6 +38,10 @@ app.use(methodOverride());
 
 app.use('/', express.static(__dirname + '/www/dist'));
 app.use('/register', express.static(__dirname + '/app/client'));
+app.use('/travel-receipts', (req, res) => {
```

Comment:
Should be `/resumes`

Reply:
Also, do we really want to do this? We could just open the dropbox link rather than redirecting like this... Seems kinda overkill

Reply:
its just what they did on theirs. and yeah i think i flipped the requests, i noticed that on the last one and im swapping them
we can just redirect i guess. we can ask chandler